solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -242,6 +263,8 @@ contract RandomBeacon is Ownable {
         dkgResultSubmissionReward = 0;
         sortitionPoolUnlockingReward = 0;
         maliciousDkgResultSlashingAmount = 50000e18;
+        sortitionPoolRewardsBanDuration = 2 weeks;
```

Comment:
Can we add a TODO to revisit it? I think it has to be longer but unsure yet, we need to think about undelegation period as well.

Reply:
See https://github.com/keep-network/keep-core/pull/2707/commits/6be521c1ffbd85c473e05aee899ab786382f4eba